### DISCUSSION TITLE
"Cannot assign requested address" error when trying to follow along with the flowkit tutorial.

### DISCUSSION THREAD

**Discussion by Mylenesch** [whitews/FlowKit#180] [Q&A]
Hi all, I am a new flowkit user and I am trying to follow along with the tutorial here. 
For context this is all the code I have:

---------------------------------------------------------------------------
!pip install flowclient
import flowclient
import os

TOKEN = "mystringtoken" #omitted the actual string 

conn = flowclient.connect(
    url=os.getenv("FLOWAPI_URL", "http://localhost:9090"),
    token=TOKEN,
    ssl_certificate=os.getenv("SSL_CERTIFICATE_FILE"),
)

query_spec = flowclient.aggregates.spatial_aggregate_spec(
    locations=flowclient.daily_location_spec(
        date="2016-01-01",
        aggregation_unit="admin3",
        method="last",
        subscriber_subset=None,
    )
)

query_id = flowclient.run_query(connection=conn, query_spec=query_spec)
query_id
---------------------------------------------------------------------------

It all works except for the part where I define the query_id. It gives me the following error:
FlowclientConnectionError: Unable to connect to FlowKit API at http://localhost:9090/: [Errno 99] Cannot assign requested address

Does someone see the mistake?

**Comment by whitews** [comment]
This repo is for the FlowKit flow cytometry library. Looks like you are using a different package.
